shell-ui/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# vim:sw=4:ts=4:et
+set -x
+CONFIG=$(sed -e ':a;N;$!ba;s/\n/ /g' /etc/metalk8s/shell-ui/config.json | sed -e 's/"/\\\\"/g')
+sed -i "s#%CONFIG%#${CONFIG}#g" "$(pwd)"/shell/solution-ui-navbar.*.js
```

Comment:
So... likely unsafe as mentioned before, and really ugly :stuck_out_tongue_closed_eyes: 
IMO, we should take the same approach as MetalK8s UI:
- bake a default configuration in the image, which uses relative paths
- mount any custom configuration over the one baked-in
- serve this configuration file using the same NGINX process, and load it client-side at runtime